tests/configs/gpii.tests.all.config.json5

```diff
@@ -43,6 +43,6 @@
         }
     },
     "mergeConfigs": [
-        "%gpii-universal/gpii/configs/gpii.config.untrusted.development.all.local.json5"
+        "%gpii-universal/gpii/configs/gpii.config.untrusted.development.manualTesting.json5"
```

Comment:
Binding to "manualTesting" is what is causing the current test failure since the db will not be correctly reprovisioned within runs. However, this exposes wider problems in the config binding across the whole project - see top-level comment.